site/src/docs/components/checkbox/customisation.mdx

```diff
@@ -16,7 +16,9 @@ export default ({ children, pageContext }) => <TabsLayout pageContext={pageConte
 </Notification>
 
 ### Customisation properties
-You can use the `style` property to customise the component. See all available theme variables in the table below.
+In order to customise how the component looks, you can either use the `style` property or add the class containing your custom styles into the `className` property.
```

Comment:
Should we also have separate instructions for React and Core here?

Reply:
True enough! This sentence for React and the new sentence for Core.